### PR TITLE
Disabled the negative features of plusplus

### PR DESCRIPTION
**Pull request by heath-freenome**
- I'm not sure how to test this... :(

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -76,6 +76,7 @@ cmake-build-*/
 
 # IntelliJ
 out/
+.idea/
```
Comment (shields-fn) [inline .gitignore]: This doesn't look like a related change?

File: plusplus/operations/slack_handler.py [hunk 2]
```diff
@@ -78,7 +78,7 @@ def process_incoming_message(event_data, req):
         team.slack_client().api_call(
```
Comment (shields-fn) [inline plusplus/operations/slack_handler.py]: You can also drop line 77 (which GitHub for some reason won't let me attach a comment to).

File: .gitignore (hunk 1 above)
Comment (heath-freenome) [reply .gitignore]: This was so I don't have to deal with all the files that IntelliJ put into that folder.  I can remove it if it feels better for you.

File: plusplus/operations/slack_handler.py (hunk 2 above)
Comment (heath-freenome) [reply plusplus/operations/slack_handler.py]: done

File: .gitignore (hunk 1 above)
Comment (shields-fn) [reply .gitignore]: The change is fine, but it should be in a separate PR.

Comment (heath-freenome) [reply .gitignore]: oops... just merged :(

**Pull request merged**